### PR TITLE
Add selectors on Group settings and info

### PR DESCRIPTION
**Pull request by comzeradd**
This would make the selection of Group settings tabs easier, especially for the selenium tests. It also add consistency with the Profile settings.

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

**Comment by johngian** [comment]
Code looks OK but I don't really like the naming format. It seems like this is the same in edit profile too.
I would prefer something like ``<tab-name>-tab``.

@comzeradd Do you think that there is too much work for that? If it is an easy fix I would suggest we use a more descriptive naming scheme both here and in the profiles.

**Comment by comzeradd** [comment]
I agree. I amended the commit. I suggest I open a different PR for the profile, because we'll have to amend the selenium tests too.

**Comment by comzeradd** [comment]
@johngian PR updated. I added a new commit with some new selector for the group information.

File: mozillians/jinja2/groups/group.html [hunk 1]
```diff
@@ -66,16 +66,24 @@ <h1 id="group_name">
           <span>
             {{ _('IRC Channel') }}:
           </span>
-          {{ group.irc_channel }}
+          <span id="group-irc">
```
Comment (johngian) [inline mozillians/jinja2/groups/group.html]: Here you are not just changing the selectors. Why do we need this ``<span>``? Are you sure that this change doesn't brake anything else in the UI?

Comment (johngian) [inline mozillians/jinja2/groups/group.html]: This also has other occurences.

Comment (comzeradd) [inline mozillians/jinja2/groups/group.html]: We need a selector to check the values of these fields. I made a tiny addition to the css so this doesn't change anything in the UI.

**Comment by johngian** [comment]
Please squash these commits to a single one.

**Comment by comzeradd** [comment]
PR updated. Commits squashed and rebased.

File: mozillians/jinja2/groups/includes/group_edit_access.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
     </div>
   </form>
 
-  <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
+  <form class="edit-group" id="form-curators" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_access.html]: Please use ``curators-form`` instead.

Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_access.html]: This also has other occurrences: use ``<section>-<element>``

File: mozillians/jinja2/groups/includes/group_edit_access.html [hunk 3]
```diff
@@ -102,7 +102,7 @@
   </form>
 
   {% if waffle.flag('group-terms-invalidation') %}
-    <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
+    <form class="edit-group" id="form-terms" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_access.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_access.html [hunk 4]
```diff
@@ -141,7 +141,7 @@
       </div>
     </form>
 
-    <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
+    <form class="edit-group" id="form-expiration" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=access">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_access.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_admin.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <div role="tabpanel" class="tab-pane fade" id="admin">
-  <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=admin">
+  <form class="edit-group" id="form-administration" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=admin">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_admin.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_description.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <div role="tabpanel" class="tab-pane fade in active" id="description">
-  <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=description" enctype="multipart/form-data">
+  <form class="edit-group" id="form-description" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=description" enctype="multipart/form-data">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_description.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_invitation.html [hunk 7]
```diff
@@ -1,5 +1,5 @@
-<div role="tabpanel" class="tab-pane fade" id="invitation">
-  <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=invitation">
+<div role="tabpanel" class="tab-pane fade" id="invitations">
+  <form class="edit-group" id="form-invitations" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=invitations">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_invitation.html [hunk 8]
```diff
@@ -50,7 +50,7 @@
     </div>
   </form>
 
-  <form class="edit-group" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=invitation">
+  <form class="edit-group" id="form-invite" method="POST" action="{{ url('groups:group_edit', group.url) }}?next=invitations">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: Same here.

File: mozillians/jinja2/groups/includes/group_edit_invitation.html [hunk 9]
```diff
@@ -31,7 +31,7 @@
                 <div class="col-sm-4">
                   <div class="invite-controls right">
                     {% set invite_pk = redeemer.groups_invited.get(group=group).pk %}
-                    <a href="{{ url('groups:delete_invite', invite_pk) }}?next=invitation">
+                    <a href="{{ url('groups:delete_invite', invite_pk) }}?next=invitations">
                       <span class="glyphicon glyphicon-remove" title="{{ _('Delete') }}" aria-hidden="true"></span>
                     </a>
                     <a href="{{ url('groups:send_invitation_email', invite_pk) }}?next=invitation">
```
Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: Fix next here.

Comment (comzeradd) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: What do you mean?

Comment (johngian) [inline mozillians/jinja2/groups/includes/group_edit_invitation.html]: You are changing the invitations sections id. You should also fix ``next=foo`` here.

**Comment by comzeradd** [comment]
PR updated

**Comment by johngian** [comment]
``Select group type is broken``
https://github.com/mozilla/mozillians/blob/master/mozillians/static/mozillians/js/group_edit.js#L10

**Comment by johngian** [comment]
When doing this kind of changes please always make sure that are references are fixed too.

**Comment by johngian** [comment]
I suggest you ``grep`` the rest of the changes too.

**Comment by comzeradd** [comment]
PR updated. This was the only occurrence of form selection in the js code and along with the invitations the only selector change in this PR. All other changes are new selectors we don't currently use in javascript. I did a local test of all forms just to be sure.

**Comment by johngian** [comment]
Code looks OK. Please do some extensive browsing to see if something gets rendered wrong.
r+wc

**Pull request merged**